cmd/client.go

```diff
@@ -196,6 +196,21 @@ func newCliCommand() *cobra.Command {
 				return errors.Annotatef(err, "fail to open PD client, pd=\"%s\"", cliPdAddr)
 			}
 			ctx := defaultContext
+			_, captureInfos, err := cdcEtcdCli.GetCaptures(ctx)
+			if err != nil {
+				return err
+			}
+			cdcClusterVer, err := version.GetTiCDCClusterVersion(captureInfos)
+			if err != nil {
+				return err
+			}
+			isUnknownVersion, err := version.CheckTiCDCClusterVersion(cdcClusterVer)
+			if err != nil {
+				return err
+			}
+			if isUnknownVersion {
+				return errors.NewNoStackError("TiCDC cluster is unknown, please start TiCDC cluster")
+			}
```

Comment:
Note this change means when we execute cli commands, TiCDC server must be alive, some points should be payed attention to
1. Some commands don't require TiCDC server alive or we need to execute some commands if TiCDC server can't start up(because of some bugs), such as `cdc cli unsafe`, what should we do in this scenario?
2. Whether some test cases are broken after this change. (Considering the scenario we create a changefeed first, and then start up TiCDC servers)

Reply:
Thanks for the remaindering, I will put the restriction to the changefeed sub-command only.

Reply:
> Whether some test cases are broken after this change. (Considering the scenario we create a changefeed first, and then start up TiCDC servers)

I think creating a changefeed is meaningless without a running TiCDC cluster. Also if the TiCDC starts too late, it is possible that TiDB GC safepoint exceeds changefeed start-ts.

Reply:
@leoppro PTAL